src/commands/idea/orteil-gamegen.ts

```diff
@@ -0,0 +1,264 @@
+  'through the seasons', 'while collecting them all', 'to work on your self-esteem', 'with no interactivity whatsoever',
+  'through the internet', 'in realtime', 'with geolocation', '(it\'s not really a game)', 'in binary', 'metaphorically', 'symbolically',
+  'repetitively', 'competitively', '(in beta)', 'in front of the whole world', 'and you gradually become beautiful',
+  'but you gradually turn into a monster', 'and you control light and darkness', 'and you can summon a variety of animals',
+  'and you can create a variety of objects', 'and you have different powers at different times',
+  'and everybody in the game speaks in broken english', 'and everybody is voiced by other players currently playing the game',
+  'and you play on the same world as everyone else', 'and near the end you ascend to godhood', 'and gravity doesn\'t exist',
+  'with astrology', 'with maths', 'with geometry', 'with quantum physics', 'and you can escape to other dimensions',
+  'and it\'s all procedurally generated', 'with terraforming', 'with the power of baking', 'with a pickaxe', 'with a shovel',
+  'with an axe', 'with a magic wand', 'and also you push around blocks', 'while telling jokes', 'but you control the hero indirectly',
+  'and you can control everyone in the game indirectly', 'with the power of javascript', 'and you learn a thing or two along the way',
+  'and the game won\'t stop scrolling', 'by reversing time', 'by controlling the flow of time',
+  'by changing the physical properties of things', 'by altering your body', 'by changing the rules of the game', 'from outside the game',
+  'by toying with gravity', 'by toying with physics', 'through social engineering', 'with meta game mechanics', 'ironically',
+  'unironically', 'randomly', 'with occasional jump-scares', 'and the game is stupidly linear', 'and you do that with only one button',
+  'but the controls are ridiculously difficult', 'and the graphics upgrade as you play', 'and the music gets progressively worse',
+  'and there\'s a laughtrack', 'and there\'s an audience that reacts to your actions', 'but you\'re the clumsiest person ever',
+  'and the sound effects are gameplay hints', 'and the gameplay changes depending on when you\'re playing it',
+  'and the gameplay changes depending on where you\'re playing it', 'and the gameplay depends on the time of the day',
+  'and the game reacts differently to how you play it', 'and the game reacts differently depending on who plays it',
+  'and it\'s sort of a parody', '- or so it seems', '- with a twist', 'except it\'s all a dream', '(it gets pretty meta)',
+  '(it gets kinda self-referential)', '(and it\'s also a commentary on video games as a media)', 'because of social differences',
+  'to win the game', 'but doing so loses the game', 'or whatever']
+));
+
+// everything
+let allNames = [];
+let allActions = [];
+let allThings = [];
+let allGoals = [];
+
+for (let i in Genres) {
```

Comment:
Why looping `for (let i in Genres)`? Could use `for (const genre of Genres)`

Reply:
code is as close as possible to source code